script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -425,8 +429,8 @@ The next thing I knew, I couldn't even breathe properly anymore.
 -- 精神活動とは切り離されているはずの<生|し><命|ん><活|ぞ><動|う>が、心に引きずられて張り裂けたかのよう。
 // *sigh* here we go again boys...
 // I guess this is all just a "you are now breathing and blinking manually" except more serious
-It was as if my vital functions,
- which are supposed to be able to operate independently of my thoughts,
+It was as if my heart,
+ something ostensibly able to operate independently of my thoughts,
  had been taken over by my mind and dashed to pieces.
```

Comment:
"dashed to pieces" doesn't fit with heart. 

>mind and was close to bursting.

Reply:
Bursting doesn't really make sense in context here to me. Also, breaking/shattering/dashing to pieces a hart is a pretty common collocation.